### ISSUE TITLE
Ubuntu mirrors.txt only ever has one entry

### ISSUE THREAD

**Issue by andrewvaughan** [Sithuk/ubuntu-server-zfsbootmenu#52]
Similar to the discussion here:
https://github.com/jblakeman/apt-select/issues/62

I'm unsure the reason, but the `mirrors.txt` file that this project uses to lookup faster alternatives for mirroring packages only has a single entry, the default archive, making the entire mirror lookup process a noop:

http://mirrors.ubuntu.com/mirrors.txt

My initial presumption was that Ubuntu is improving their edge-serving capability to reduce the need for mirrors; however, entries like this still exist:

http://mirrors.ubuntu.com/US.txt

It may be worth looking into why this is the case, but FYI that the script doesn't perform the way expected, at the moment, due to this.

**Comment by Sithuk** [comment]
I noticed the issue in my recent testing. mirrors.ubuntu.com was working until relatively recently. I've created a thread on the ubuntu forums, but no one has explained whether the issue is a deprecation or a temporary issue.
https://ubuntuforums.org/showthread.php?t=2498931

There is an alternative approach for mirrors that uses the following site, but I would need to find a way to quickly scan the approximately 500 entries to rank them.
https://launchpad.net/ubuntu/+archivemirrors
